manifest.json

```diff
@@ -18,12 +18,12 @@
 		"email": "maniackc_dev@crudelis.fr"
 	}],
     "requirements": {
-        "yunohost": ">= 3.6"
+        "yunohost": ">= 3.8.1"
     },
 	"multi_instance": true,
 	"services": [
 		"nginx",
-		"php7.0-fpm",
+		"php7.3-fpm",
```

Comment:
The CI is not happy with this... 
✘ [YEP-2.1?] php7.3-fpm service not installed by the install file but present in the manifest

Reply:
Hmm yeah probably before 4.0 the CI is not happy. Let's revert this line